lib/id3c/api/routes.py

```diff
@@ -149,14 +150,56 @@ def receive_fhir(*, session):
 
     return "", 204
 
+
+@api_v1.route("/receiving/manifest", methods = ['POST'])
+@content_types_accepted(["application/json"])
+@check_content_length
+@authenticated_datastore_session_required
+def receive_manifest(*, session):
+    """
+    Receive JSON representation of manifest resources.
+    """
+    document = request.get_data(as_text = True)
+
+    LOG.debug(f"Received Manifest Document")
+
+    add_provenance(document, request)
+    datastore.store_manifest(session, document)
+
+    return "", 204
```

Comment:
If there's no immediate use case for this endpoint, may be better to leave it out.